README.md

```diff
@@ -42,6 +42,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
     <td align="center"><a href="https://github.com/xche529"><img src="https://avatars.githubusercontent.com/u/126313171?s=400&v=4" width="140px;" alt=""/><br /><sub><b>Tommy Chen</b></sub></a><br /><a href="https://github.com/lucas2005gao/REACT Template/commits?author=xche529" title="Code">💻</a></td>
     <td align="center"><a href="https://github.com/codecreator127"><img src="https://avatars.githubusercontent.com/u/120153300?s=400&v=4" width="140px;" alt=""/><br /><sub><b>John Lin</b></sub></a><br /><a href="https://github.com/lucas2005gao/REACT Template/commits?author=djos192" title="Code">💻</a></td>
     <td align="center"><a href="https://github.com/jacobmathew105"><img src="https://avatars.githubusercontent.com/u/136278107?s=400&v=4" width="140px;" alt=""/><br /><sub><b>Jacob Mathew</b></sub></a><br /><a href="https://github.com/lucas2005gao/REACT Template/commits?author=jacobmathew105" title="Code">💻</a></td>
+    <td align="center"><a href="https://github.com/sbeen56"><img src="https://avatars.githubusercontent.com/u/161543207?s=400&v=4" width="140px;" alt=""/><br /><sub><b>Subeen Ban</b></sub></a><br /><a href="https://github.com/lucas2005gao/REACT Template/commits?author=sbeen56" title="Code">💻</a></td>

```

Comment:
You've added your avatar to the same row as everyone else, but as that row is already full (5 people in it) it causes everyone's profile to shrink/resize to accommodate this. You can easily fix it by putting this code in the row above (i.e. next to my avatar). 😄 
Attached are images for comparison
![image](https://github.com/UoaWDCC/VPS/assets/127293604/3c53d48c-0c36-49bb-b8e1-5e851f99c4f8)
![image](https://github.com/UoaWDCC/VPS/assets/127293604/ec70e65a-76c6-4656-86a2-ea1171267ccd)